Cargo.lock

```diff
@@ -2552,16 +2559,15 @@ checksum = "4756f7db3f7b5574938c3eb1c117038b8e07f95ee6718c0efad4ac21508f1efd"
 
 [[package]]
 name = "zip"
-version = "0.6.3"
+version = "0.6.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "537ce7411d25e54e8ae21a7ce0b15840e7bfcff15b51d697ec3266cc76bdf080"
+checksum = "e6fa4aa90e99fb8d701bda16fb040d8ed2f9c7176fb44de750e880a74b580315"
```

Comment:
@notmandatory  I have manually pinned the version, and put it into cargo.lock.. Do we need to do a `patch-crates.io` too in cargo.toml?

Reply:
Also I'm pretty sure we can't publish a crate if there's a `patch-crates.io` section in our cargo.toml.

Reply:
Okay.. Noted.. Will do it this way..

Reply:
I rebased and made the changes. Should work now, after CI finishes I'll merge it. :-)